### PR TITLE
Add get aws client alternative url from env vars

### PR DESCRIPTION
**Pull request by mabe13**

### CODE REVIEW

**Comment by JackPlowman** [comment]
Data-to-file-conversion is now working up to the script to extract the database due to the database not being accessible yet

File: application/data_to_file_conversion/AwsClient.py [hunk 1]
```diff
@@ -10,6 +10,11 @@ class Factory:
     use_ssl = True
     logger = logging.getLogger("diagnostics")
 
+    def __init__(self):
```
Comment (stefaniuk) [inline application/data_to_file_conversion/AwsClient.py:13]: Shouldn't we always be using SSL?

File: build/automation/var/profile/local.mk [hunk 2]
```diff
@@ -12,3 +12,10 @@ TEST_MODE:= True
 
 # ==============================================================================
 # Infrastructure variables
+
+AWS_USE_SSL:= False
+AWS_ALT_SECRETSMANAGER_URL := http://docker.for.mac.localhost:4584
+AWS_ALT_S3_URL := http://docker.for.mac.localhost:4572
```
Comment (stefaniuk) [inline build/automation/var/profile/local.mk:18]: Can we put a comment in the TODO.md file to check if we can use the Docker alias to refer to localstack?

File: application/data_to_file_conversion/AwsClient.py (hunk 1 above)
Comment (JackPlowman) [reply application/data_to_file_conversion/AwsClient.py]: SSL is on by default so doesn't need to be set

File: build/automation/var/profile/local.mk (hunk 2 above)
Comment (mabe13) [reply build/automation/var/profile/local.mk:17]: @stefaniuk @JackPlowman Wonder if we could add a docker localstack alias by incorporate the localstack setup as part of the docker compose file (as it's only used for local development) ?

Comment (stefaniuk) [reply build/automation/var/profile/local.mk:17]: That's not a bad idea, Matt.

**Pull request merged**